common/string_calls.c

```diff
@@ -139,6 +139,44 @@ g_text2bool(const char *s)
     return 0;
 }
 
+/*****************************************************************************/
+char *
+g_strnjoin(char *dest, int dest_len, const char *joiner, const char *src[], int src_len)
```

Comment:
src_len == 0 causes src[0]to be dereferenced at line 170. Probably need a guard for this (and a test?)

Reply:
fixed